Content.Shared/Polymorph/Components/ChameleonProjectorComponent.cs

```diff
@@ -54,6 +57,12 @@ public sealed partial class ChameleonProjectorComponent : Component
     [DataField]
     public float MaxHealth = 100f;
 
+    /// <summary>
+    /// Popup shown to the user when they try to disguise as an entity inside a container.
+    /// </summary>
+    [DataField]
+    public LocId ContainerPopup = "chameleon-projector-inside-container";
```

Comment:
Seems a bit strange to have the messages set in the component, unless they are intended to change across prototypes

Reply:
i think its just good practice to have the locids on the component instead of hardcoded in the system